spec/fixtures/proofing/lexis_nexis/instant_verify/year_of_birth_fail_response.json

```diff
@@ -48,10 +48,6 @@
           "ItemName": "DOBYearVerified",
           "ItemStatus": "pass"
         },
-        {
-          "ItemName": "SSNLowIssuance",
-          "ItemStatus": "pass"
-        },
```

Comment:
is this just a true-up? not strictly related to DOB right?

Reply:
Yeah, this is not longer in the response